Lib/asyncio/base_tasks.py

```diff
@@ -74,3 +76,51 @@ def _task_print_stack(task, limit, file):
     if exc is not None:
         for line in traceback.format_exception_only(exc.__class__, exc):
             print(line, file=file, end='')
+
+
+# WeakKeyDictionary of {Task: EventLoop} containing all tasks alive.
+_all_tasks = weakref.WeakKeyDictionary()
+
+# Dictionary containing tasks that are currently active in
+# all running event loops.  {EventLoop: Task}
+_current_tasks = {}
+
+
+def current_task(loop=None):
+    if loop is None:
+        loop = events.get_event_loop()
```

Comment:
Use `events.get_running_loop()` here -- we want to use the new API here.  It doesn't make sense to call `asyncio.current_task()` and accidentally create a new event loop because `get_event_loop()` thinks so.